src/main/java/com/github/cameltooling/lsp/internal/instancemodel/OptionParamURIInstance.java

```diff
@@ -102,4 +111,15 @@ public String getDescription(ComponentModel componentModel, KameletsCatalogManag
 	public CamelURIInstance getCamelUriInstance() {
 		return camelURIInstance;
 	}
+	
+	private JSONSchemaProps getKameletPropertyByKeyName(KameletsCatalogManager kameletCatalogManager, String keyName) {
+		List<Kamelet> kamelets = kameletCatalogManager.getCatalog().getKameletsByName(this.camelURIInstance.getComponentAndPathUriElementInstance().getApiNamePath().getValue());
+		if (kamelets.size()>0) {
+			Kamelet kamelet = kamelets.get(0);
+			if (kamelet.getSpec().getDefinition().getProperties().containsKey(keyName)) {
+				return kamelet.getSpec().getDefinition().getProperties().get(keyName);
+			}
```

Comment:
I think that we can directly return the result:

```suggestion
			return kamelet.getSpec().getDefinition().getProperties().get(keyName);
```